src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -61,9 +60,9 @@ def get(self, request: Request, organization: Organization) -> Response:
         result = ReplayDataSourceValidator(data=request.GET)
         if not result.is_valid():
             raise ParseError(result.errors)
-        data_source = Dataset.Discover
-        if result.validated_data["data_source"] == Dataset.IssuePlatform:
-            data_source = Dataset.IssuePlatform
+        data_source = "discover"
+        if result.validated_data["data_source"] == "issue_platform":
+            data_source = "issue_platform"
         try:
             replay_counts = get_replay_counts(
                 snuba_params, request.GET.get("query"), request.GET.get("returnIds"), data_source
```

Comment:
would be ideal if data_source parameter to `get_replay_counts` were the enum value itself instead of a string.